### PR TITLE
ADD: Added Wio Terminal's functional pins and LCD driver to board init

### PR DESCRIPTION
**Pull request by ansonhe97**
- Added Wio Termnal's functional pins such as buttons, switches, Light sensor and so on.
- Added LCD driver to board init.

### CODE REVIEW

File: ports/atmel-samd/boards/seeeduino_wio_terminal/pins.c [hunk 1]
```diff
@@ -33,9 +33,10 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     { MP_ROM_QSTR(MP_QSTR_RX), MP_ROM_PTR(&pin_PB27) },
 
     // SPI pins
-    { MP_ROM_QSTR(MP_QSTR_MOSI), MP_ROM_PTR(&pin_PA02) },
-    { MP_ROM_QSTR(MP_QSTR_SCK), MP_ROM_PTR(&pin_PA03) },
-    { MP_ROM_QSTR(MP_QSTR_MISO), MP_ROM_PTR(&pin_PA00) },
+    { MP_ROM_QSTR(MP_QSTR_MOSI), MP_ROM_PTR(&pin_PB02) },
+    { MP_ROM_QSTR(MP_QSTR_SCK), MP_ROM_PTR(&pin_PB03) },
+    { MP_ROM_QSTR(MP_QSTR_MISO), MP_ROM_PTR(&pin_PB00) },
+    { MP_ROM_QSTR(MP_QSTR_SS), MP_ROM_PTR(&pin_PB01) },
```
Comment (tannewt) [inline ports/atmel-samd/boards/seeeduino_wio_terminal/pins.c:39]:
```suggestion
    { MP_ROM_QSTR(MP_QSTR_CS), MP_ROM_PTR(&pin_PB01) },
```

We normally call it CS for chip select and we're trying to move away from the term `slave` as well. https://circuitpython.readthedocs.io/en/latest/docs/design_guide.html#terminology

File: ports/atmel-samd/boards/seeeduino_wio_terminal/pins.c [hunk 2]
```diff
@@ -44,6 +45,43 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
     // LED pins
     { MP_ROM_QSTR(MP_QSTR_LED),   MP_ROM_PTR(&pin_PA15) }, // status
 
+    // LCD Display
+    { MP_OBJ_NEW_QSTR(MP_QSTR_TFT_MISO),  MP_ROM_PTR(&pin_PB18) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_TFT_MOSI),  MP_ROM_PTR(&pin_PB19) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_TFT_SCK),  MP_ROM_PTR(&pin_PB20) },
+    { MP_ROM_QSTR(MP_QSTR_TFT_SS),  MP_ROM_PTR(&pin_PB21) },
```
Comment (tannewt) [inline ports/atmel-samd/boards/seeeduino_wio_terminal/pins.c:52]:
```suggestion
    { MP_ROM_QSTR(MP_QSTR_TFT_CS),  MP_ROM_PTR(&pin_PB21) },
```

File: ports/atmel-samd/boards/seeeduino_wio_terminal/pins.c [hunk 3]
```diff
@@ -44,6 +45,43 @@ STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
...
+    { MP_ROM_QSTR(MP_QSTR_TFT_DC),  MP_ROM_PTR(&pin_PC06) },
+    { MP_ROM_QSTR(MP_QSTR_TFT_BACKLIGHT),  MP_ROM_PTR(&pin_PC05) },
+    { MP_ROM_QSTR(MP_QSTR_TFT_RESET),  MP_ROM_PTR(&pin_PC07) },
+
+    // SD Card
+    { MP_OBJ_NEW_QSTR(MP_QSTR_SD_MISO),  MP_ROM_PTR(&pin_PC18) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_SD_MOSI),  MP_ROM_PTR(&pin_PC16) },
+    { MP_OBJ_NEW_QSTR(MP_QSTR_SD_SCK),  MP_ROM_PTR(&pin_PC17) },
+    { MP_ROM_QSTR(MP_QSTR_SD_SS),  MP_ROM_PTR(&pin_PC19) },
```
Comment (tannewt) [inline ports/atmel-samd/boards/seeeduino_wio_terminal/pins.c:61]:
```suggestion
    { MP_ROM_QSTR(MP_QSTR_SD_CS),  MP_ROM_PTR(&pin_PC19) },
```

**Comment by ansonhe97** [comment]
Just made the suggested changes :)

**Pull request merged**